Lib/random.py

```diff
@@ -238,19 +238,19 @@ def __init_subclass__(cls, /, **kwargs):
                 break
 
     def _randbelow_with_getrandbits(self, n):
-        "Return a random int in the range [0,n).  Returns 0 if n==0."
+        "Return a random int in the range [0,n).  Returns 0 if n<=1."
 
-        if not n:
+        if n <= 1:
```

Comment:
This change is also unnecessary, and likely slows down the common case a tiny amount. `getrandbits(0)` works in master.

Reply:
I just tested this for python3.9 on Debian testing and `if n:` is indeed faster than the alternatives here;

```
$ python3 -m timeit -s 'i = 1234' 'if i: pass'
10000000 loops, best of 5: 38.3 nsec per loop

$ python3 -m timeit -s 'i = 1234' 'if i==0: pass'
5000000 loops, best of 5: 56.5 nsec per loop

$ python3 -m timeit -s 'i = 1234' 'if i<=1: pass'
5000000 loops, best of 5: 59.2 nsec per loop
```
I'll change this back.